Scheduler - Getting Started: Add ASP.NET Core Snippets (#9038)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/15 Edit Appointments.md

#include common-tutorialbutton-named with { url: "/Documentation/Guide/UI_Components/Scheduler/Appointments/Add_Appointments/", name: "Add Appointments" }
#include common-tutorialbutton-named with { url: "/Documentation/Guide/UI_Components/Scheduler/Appointments/Update_Appointments/", name: "Update Appointments" }
#include common-tutorialbutton-named with { url: "/Documentation/Guide/UI_Components/Scheduler/Appointments/Delete_Appointments/", name: "Delete Appointments" }

> Scheduler allows users to add, update, and delete appointments. All edit operations are enabled in the default component configuration. To disable specific operations, configure [editing](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/editing '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/') properties.

Reviewer: ```suggestion
Scheduler allows users to add, update, and delete appointments. All editing operations are enabled by default. To disable specific operations, configure the [editing](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/editing '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/') properties.
```

            <DxEditing
                :allow-dragging="false"
            />
        </DxScheduler>
    </template> 

    <script setup lang="ts">
    import { DxScheduler, DxEditing } from 'devextreme-vue/scheduler';

    </script>

##### React 

    <!-- tab: App.tsx -->
    import { Scheduler, Editing } from 'devextreme-react/scheduler';
    // ...

    export default function App() {
        return (
            <Scheduler>
                <Editing
                    allowDragging={false}
                />
            </Scheduler>
        );
    }

---

When users add, update, or delete appointments, Scheduler calls the following event handlers:

- [onAppointmentAdding](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/onAppointmentAdding.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentAdding/')
- [onAppointmentUpdating](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/onAppointmentUpdating.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentUpdating/')
- [onAppointmentDeleting](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentDeleting)

> You can use these handlers to extend Scheduler's functionality. The following example configures **onAppointmentAdding** and **onAppointmentUpdating** to ensure recurring appointments do not share cells with other appointments:

Reviewer: ```suggestion
You can use these handlers to extend Scheduler's functionality. The following example configures **onAppointmentAdding** and **onAppointmentUpdating** to prevent recurring appointments from sharing cells with other appointments:
```
